3-platform/order/infra/src/main/java/com/_6core/platform/orderinfra/adapter/driven/persistence/entity/OrderEntity.java

```diff
@@ -0,0 +1,31 @@
+package com._6core.platform.orderinfra.adapter.driven.persistence.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigInteger;
+import java.util.Set;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@Entity
+@Table(name = "orders")
+public class OrderEntity {
+  @Id private String id;
+  private BigInteger total;
+
+  @Enumerated(value = EnumType.STRING)
+  private OrderStatus status;
+
+  @OneToMany(mappedBy = "order")
+  private Set<OrderItemEntity> orderItems;
+
+  public enum OrderStatus {
+    UNDEFINED
+  }
```

Comment:
I hate using inner classes/ Move this enum to the `domain layer`. 
regarding inner classes:  it is tricky questions when to use inner classes because of Single Responsibility Principle violation. 
 I see only one usage of nested/inner classes as `container` for outer class only. e.g. how it implemented with Node<> and LinkedList<>
 
```suggestion
  public enum OrderStatus {
    UNDEFINED
  }
```

Reply:
ok